src/operon.ts

```diff
@@ -388,11 +392,21 @@ export class Operon {
   }
 
   /**
-   * A recovery process that runs during Operon init time.
+   * A recovery process that by default runs during Operon init time.
    * It runs to completion all pending workflows that were executing when the previous executor failed.
    */
-  async recoverPendingWorkflows() {
-    const pendingWorkflows = await this.systemDatabase.getPendingWorkflows();
+  async recoverPendingWorkflows(executorIDs?: string[]): Promise<WorkflowHandle<any>[]> {
```

Comment:
Isn't it possible to make `["local]` a default argument?

Reply:
Done!